Arm/Insts/DPI/Bitfield.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Shilpi Goel, Nevine
+-/
+-- For now, support only UBFM (immediate) aliased as LSR (immediate):
+-- 32- and 64-bit versions
+
+import Arm.Decode
+import Arm.Insts.Common
+
+namespace DPI
+
+open Std.BitVec
+
+@[simp]
+def exec_bitfield (inst: Bitfield_cls) (s : ArmState) : ArmState :=
+  -- TODO: match opc
```

Comment:
I'd suggest matching on `opc` and if it isn't equal to `0b10` (UBFM's), throwing an "unimplemented instruction" error (here's an example: https://github.com/leanprover/LNSym/blob/3fc5b4fc614bc4eb954cf65e60949744b96f15b9/Arm/Insts/DPR/Conditional_select.lean#L32)